blt/src/CgovNpmTrait.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Cgov;
+
+use Cgov\Task\Npm\CleanInstall;
+use Cgov\Task\Npm\RunScript;
+use Robo\Task\Npm\Install;
+
+/**
+ * Trait for NPM helper tasks.
+ *
+ * @package Acquia\Blt\Custom
+ */
+trait CgovNpmTrait {
+
+  /**
+   * Creates a task to run an npm install.
+   *
+   * @param null|string $pathToNpm
+   *   Path to a specific NPM installation.
+   *
+   * @return \Robo\Collection\CollectionBuilder
+   *   CollectionBuilder for executing the npm install task.
+   */
+  protected function taskNpmInstall($pathToNpm = NULL) {
+    return $this->task(Install::class, $pathToNpm);
+  }
+
+  /**
+   * Creates a task to run an npm clean install.
+   *
+   * @param null|string $pathToNpm
+   *   Path to a specific NPM installation.
+   *
+   * @return \Robo\Collection\CollectionBuilder
+   *   CollectionBuilder for executing the npm ci task.
+   */
+  protected function taskNpmCleanInstall($pathToNpm = NULL) {
+    return $this->task(CleanInstall::class, $pathToNpm);
+  }
+
+  /**
+   * Creates a task to run an npm script.
+   *
+   * @param null|string $pathToNpm
+   *   Path to a specific NPM installation.
+   *
+   * @return \Robo\Collection\CollectionBuilder
+   *   CollectionBuilder for executing the npm run-script task.
+   */
+  protected function taskNpmRunScript($pathToNpm = NULL) {
+    return $this->task(RunScript::class, $pathToNpm);
```

Comment:
Since I am noting that you should be taking in the tests to run, you should probably allow this to take parameters. You can do `npm run foo -- --bar --baz`. The "empty" `--` option just passes along the parameters to the script it is going to run. See https://docs.npmjs.com/cli/v9/commands/npm-run-script.

This would allso you to vary the threads, etc.

Reply:
Yep, that's what I ended up with.